Update ChatServer.java

dimits-ts opened the pull request.

Commit:
Update ChatServer.java

Commit:
Update Server.java

Screen is now static

Commit:
Update GameServer.java

Commit:
Update GameEngine.java

Minor correction on JFrame title

AAAlex-123 reviewed (approved):
k nice

AAAlex-123 committed:
removed a \n character

gameConnected and chatConnected should be in the same line, so the documentation above applies to both. Hover over them in Eclipse to see the result

AAAlex-123 commented:
wait no don't merge yet, some stuff to be fixed :p

AAAlex-123 commented:
Game ending doesn't work properly, *seriously* don't merge xd

edit: it sometimes work lmaooo

AAAlex-123 committed:
improved log, added fancy font

also setupScreen is now static because Eclipse won't stop whining about it

AAAlex-123 committed:
gameui pushMessage works

String.format go brrrrrr

AAAlex-123 committed:
gameui checkmark works

ah read the comment, the spaghetti returns

AAAlex-123 committed:
gameengine works with checkmark

look at gameui if you think it's that simple xd

AAAlex-123 commented:
ok i think it's ready. play a game to test and if ok, merge

AAAlex-123 committed:
GameBoard visibility reduction

public -> default
EMPTY is private
minor changes

AAAlex-123 committed:
GameUI scope reduced (+other minor changes)

HEIGHT_MULTIPLIER, SCREEN_WIDTH, SCREEN_HEIGHT are not static
re-ordered some methods to make more sense, added comments between sections
public -> <default>
class is final
other minor changes

AAAlex-123 committed:
GameEngine visibility reduction + small fixes

improved exit()
removed/added comments
game loop to be improved kinda maybe (?)

AAAlex-123 committed:
GameServer visibility reduction + small changes

window.dispose() instead of window.setVisible(false)
chatServer.setScreen() call in constructor not needed

AAAlex-123 committed:
GameServer doc fix

<code> to {@code}

AAAlex-123 committed:
server visibility reduction + small changes

<code> -> {@code}

AAAlex-123 committed:
server fix newline

\n to %n

AAAlex-123 committed:
logging fix newline

\n -> %n

AAAlex-123 committed:
gameui jlabel fix

AAAlex-123 committed:
xd

Commit:
Merge pull request #29 from AAAlex-123/code-formatting-visibility-final-static-changes

Code formatting visibility final static changes
